packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -76,12 +97,56 @@ const DocTemplate = ({ packageName, readme, examples, propsTables, ...customProp
     id += 1;
   }
 
+  // Both of these props are needed to be able to generate the source path
+  const showSrcPath = repositoryURL && (packageName || srcPath);
+
+  if (showSrcPath) {
+    // The repository url can be of the format 'provider:user/repository' so process those for the website
```

Comment:
Where did we get these formats from?

Reply:
I found them at https://docs.npmjs.com/files/package.json#repository